### PR TITLE
Update RouteEntry Resource RouteEntryName Field

### PR DESCRIPTION
**Pull request by linxudong7**
TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudRouteEntry -timeout=300m
=== RUN   TestAccAlicloudRouteEntryInstance
--- PASS: TestAccAlicloudRouteEntryInstance (139.57s)
=== RUN   TestAccAlicloudRouteEntryInterface
--- PASS: TestAccAlicloudRouteEntryInterface (32.55s)
=== RUN   TestAccAlicloudRouteEntryNatGateway
--- PASS: TestAccAlicloudRouteEntryNatGateway (46.19s)
=== RUN   TestAccAlicloudRouteEntryMulti
--- PASS: TestAccAlicloudRouteEntryMulti (70.79s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	289.106s

### CODE REVIEW

File: alicloud/resource_alicloud_route_entry_test.go [hunk 1]
```diff
@@ -248,6 +248,7 @@ resource "alicloud_route_entry" "default" {
 	destination_cidrblock = "172.11.1.1/32"
 	nexthop_type = "Instance"
 	nexthop_id = "${alicloud_instance.default.id}"
+	name = "${var.name}"
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_route_entry_test.go]: check name值

File: alicloud/resource_alicloud_route_entry_test.go [hunk 2]
```diff
@@ -309,6 +310,7 @@ resource "alicloud_route_entry" "default" {
   destination_cidrblock = "172.11.1.1/32"
   nexthop_type = "RouterInterface"
   nexthop_id = "${alicloud_router_interface.default.id}"
+  name = "${var.name}"
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_route_entry_test.go]: 同上

File: alicloud/resource_alicloud_route_entry_test.go [hunk 3]
```diff
@@ -345,6 +347,7 @@ resource "alicloud_route_entry" "default" {
   destination_cidrblock = "172.11.1.1/32"
   nexthop_type = "NatGateway"
   nexthop_id = "${alicloud_nat_gateway.default.id}"
+  name = "${var.name}"
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_route_entry_test.go]: 同上

File: alicloud/resource_alicloud_route_entry_test.go [hunk 4]
```diff
@@ -386,6 +389,7 @@ resource "alicloud_route_entry" "default" {
 	destination_cidrblock = "172.16.${count.index}.0/24"
 	nexthop_type = "NetworkInterface"
 	nexthop_id = "${alicloud_network_interface.default.id}"
+	name = "${var.name}"
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_route_entry_test.go]: 同上

File: alicloud/resource_alicloud_route_entry_test.go (hunk 1 above)
Comment (linxudong7) [reply alicloud/resource_alicloud_route_entry_test.go]: done

**Pull request merged**